src/pages/Competition/Person/index.tsx

```diff
@@ -102,7 +115,7 @@ export default function Person() {
           </thead>
           <tbody>
             {scheduleDays.map(({ date, dateParts }) => (
-              <>
+              <Fragment key={date}>
```

Comment:
I included this change because it removes a warning in development that all elements from a collection should have a `key`.